installer/templates/phx_assets/app.css

```diff
@@ -1,22 +1,29 @@
-@import "tailwindcss/base";
-@import "tailwindcss/components";
-@import "tailwindcss/utilities";
+/* See the Tailwind configuration guide for advanced usage
+   https://tailwindcss.com/docs/configuration */
+
+@import "tailwindcss" source(none);
+@source "../css";
+@source "../js";
+@source "../../lib/<%= @lib_web_name || @app_name %>";
+@plugin "../tailwind_heroicons.js";
+
+@plugin "../vendor/daisyui" {
+  /* In practice, you will likely ship only with two themes, as below: */
+  /* themes: light --default, dark --prefersdark; */
+  themes: all;
+}
+
+@custom-variant phx-click-loading ([".phx-click-loading&", ".phx-click-loading &"]);
+@custom-variant phx-submit-loading ([".phx-submit-loading&", ".phx-submit-loading &"]);
+@custom-variant phx-change-loading ([".phx-change-loading&", ".phx-change-loading &"]);
+@theme {
+  --color-brand: oklch(66.64% 0.2202 37.57);
```

Comment:
I believe we no longer use --color-brand and this should be removed (at least I did my best to remove it, as it does not work across daisy themes).